src/motion-queue.h

```diff
@@ -118,6 +118,10 @@ class MotionQueue {
   // The return parameter head_item_progress is set to the number
   // of not yet executed loops in the item currenly being executed.
   virtual int GetPendingElements(uint32_t *head_item_progress) = 0;
+
+  // Perform an immediate reset of the queue,
+  // even if motors are still moving.
+  virtual bool Clear() = 0;
```

Comment:
The 'even if motors are still moving': does that mean best-effort to stop motors ASAP ? Then this should be mentioned here (Imagine someone implementing it: they read the comment and need to come up with an implementation that matches that expectation).
What does the return value mean ?